### DISCUSSION TITLE
Registering JuliaPOMDP Package

### DISCUSSION THREAD

**Discussion by Omastto1** [JuliaPOMDP/POMDPs.jl#345] [Q&A] [answered]
Hi,
I am implementing a couple of new repositories and sooner or later, those would need to be registered into some registry, but I did not find any info as to how to do that in the `POMDPs.jl` wiki.

From my knowledge, I have two possibilities where to register my repository. Either Julia's `General` repository or `JuliaPOMDP repository`.

I can register the repository in `General` by tagging JuliaRegistrator with `register()`. How can I register the repository in `JuliaPOMDPRegistry`? Do I have to manually create PR with necessary files (`Compat.toml`, `Deps.toml`, `Package.toml`, `Versions.toml``)? Can I somehow generate those files?

I suppose our priority is using JuliaPOMDPRegistry, correct?

Can you also give an insight into how do registries work internally? Does `General` registry know about `JuliaPOMDPRegistry`?

**Comment by zsunberg** [comment, marked as answer]
The JuliaPOMDP registry is no longer used. Please just use General.
